libcni/api.go

```diff
@@ -632,6 +633,9 @@ func (c *CNIConfig) validatePlugin(ctx context.Context, pluginName, expectedVers
 	if err != nil {
 		return err
 	}
+	if expectedVersion == "" {
+		expectedVersion = types020.ImplementedSpecVersion
```

Comment:
According to the suggestion in [https://github.com/containernetworking/cni/issues/720#issuecomment-540046432](https://github.com/containernetworking/cni/issues/720#issuecomment-540046432), I think `expectedVersion` here should be 0.1.0 and 0.2.0 .

Reply:
@mars1024 @mccv1r0 I think we should probably hardcode this to "0.1.0" which is the closest version to "" that we have.  Effectively "" and 0.1.0 is equivalent.